packages/adapter-node/src/index.js

```diff
@@ -1,6 +1,8 @@
 // TODO hardcoding the relative location makes this brittle
-import { init, render } from '../output/server/app.js'; // eslint-disable-line import/no-unresolved
-import { path, host, port } from './env.js'; // eslint-disable-line import/no-unresolved
+// @ts-ignore
```

Comment:
any idea why this is needed here, but not in `adapter-cloudflare-workers/files/entry.js`?

Reply:
I was also scratching my head for a while trying to figure out this and ended up just ignoring this one, turns out the tsconfig for all the other serverless adapters weren't including their "source" files as well

This should just be a reminder though and will hopefully be resolved in the upcoming PR